app/templates/home.ejs

```diff
@@ -31,7 +31,7 @@
 
         </div>
 
-        <script src="https://elements.cronofy.com/js/CronofyElements.v1.32.0.js"></script>
+        <script src="https://elements.cronofy.com/js/CronofyElements.v1.34.8.js"></script>
```

Comment:
```suggestion
        <script src="https://elements.cronofy.com/js/CronofyElements.v1.35.4.js"></script>
```
Not critical to update this but might as well since we're now one minor version up.